jsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
+  "compilerOptions": {
```

Comment:
I'm not sure this file really needs to be in this PR. It's useful but has nothing to do with this particular feature. I think this should be added in a follow-up PR.

Reply:
Not sure why this is there or what it does... Probably got created when I was trying to set up the project/IDE. Will remove from this PR. Not really sure what it does but I can add it in a follow up PR if It would be useful.

Reply:
Fixed in 86f0fa3e
Follow up PR: #1331

Reply:
@emiljoha it's used by VSCode to configure the built-in JS language service. See here for details: https://code.visualstudio.com/docs/languages/jsconfig

Reply:
Ah! That makes a lot of sense. Thanks!